profileservice/api/controllers/configs.py

```diff
@@ -22,17 +22,19 @@
 # this has to be have the file path to the folder for saving image files and other necessary files from rest service
 PROFILE_REST_STORAGE = os.getenv('REST_STORAGE', '/usr/src/app/rest')
 MONGO_PROFILE_URL = os.getenv('MONGO_PROFILE_URL', 'localhost:27017')
-MONGO_PII_URL = os.getenv('MONGO_PII_URL', 'localhost:27017')
+MONGO_PII_URL = os.getenv('MONGO_PROFILE_URL', 'localhost:27017')
 FLASK_APP = os.getenv('FLASK_APP', 'profile_rest_service')
 FLASK_ENV = os.getenv('FLASK_ENV', 'production')
 API_LOC = os.getenv('API_LOC', '../')
 PROFILE_URL_PREFIX = os.getenv('PROFILE_URL_PREFIX', '')
+SHIB_HOST = os.getenv('SHIBBOLETH_HOST', '')
+AUTH_ISSUER = os.getenv('AUTH_ISSUER', '')
 DEBUG = bool(os.getenv('DEBUG', 'False') == 'True')
 
-PROFILE_DB_NAME = 'profiledb'
-PII_DB_NAME = 'piidb'
-PROFILE_DB_PROFILE_COLL_NAME = 'NonPiiDataset'
-PII_DB_PII_COLL_NAME = 'PiiDataset'
+PROFILE_DB_NAME = os.getenv('PROFILE_DB_NAME', 'profiledb')
+PII_DB_NAME = os.getenv('PROFILE_DB_NAME', 'profiledb')
```

Comment:
```suggestion
PII_DB_NAME = os.getenv('PROFILE_DB_NAME', 'piidb')
```
This is the currently used PII database name in the Illinois instance.